libs/designer-ui/src/lib/editor/monaco/index.tsx

```diff
@@ -115,21 +117,21 @@ export const MonacoEditor = forwardRef<editor.IStandaloneCodeEditor, MonacoProps
     const [canRender, setCanRender] = useState(false);
     const currentRef = useRef<editor.IStandaloneCodeEditor>();
 
-    const initTemplateLanguage = async () => {
-      const { languages, editor } = await loader.init();
-      if (!languages.getLanguages().some((lang: any) => lang.id === Constants.LANGUAGE_NAMES.WORKFLOW)) {
-        registerWorkflowLanguageProviders(languages, editor);
-      }
-      setCanRender(true);
-    };
-
     useEffect(() => {
+      const initTemplateLanguage = async () => {
```

Comment:
Why not just add the implementation directly on Line 130 ?

Reply:
It is a dependency of the useEffect and registerWorkflowLanguageProviders is what initializes the template functions, I can however, move initTemplateLanguage into usecallback